src/views/virtualmachines/details/tabs/overview/components/VirtualMachinesOverviewTabDetails/VirtualMachinesOverviewTabDetails.tsx

```diff
@@ -115,7 +125,18 @@ const VirtualMachinesOverviewTabDetails: React.FC<VirtualMachinesOverviewTabDeta
                 <DescriptionListGroup>
                   <DescriptionListTerm>{t('Template')}</DescriptionListTerm>
                   <DescriptionListDescription data-test-id="virtual-machine-overview-details-template">
-                    {vm?.metadata?.labels?.['vm.kubevirt.io/template'] || '-'}
+                    {templateName ? (
```

Comment:
```suggestion
                    {templateName && templateNamespace ? (
```

Reply:
you should also take care of the option of name without namespace, then just show the name without a link